src/App/Pages/Vault/CipherDetailsPage.xaml

```diff
@@ -186,11 +192,11 @@
                                     AutomationProperties.IsInAccessibleTree="True"
                                     AutomationProperties.Name="{u:I18n CopyPassword}"
                                     IsVisible="{Binding Cipher.ViewPassword}"
-                                    AutomationId="LoginCopyPasswordButton" />
+                                    AutomationId="CopyValueButton" />
                             </Grid>
                             <BoxView StyleClass="box-row-separator"
                                      IsVisible="{Binding Cipher.Login.Password, Converter={StaticResource stringHasValue}}" />
-                            <Grid StyleClass="box-row" IsVisible="{Binding ShowTotp}">
+                            <Grid StyleClass="box-row" IsVisible="{Binding ShowTotp}" AutomationId="ItemRow">
```

Comment:
⛏️ I'm not a fan of more than 1 or 2 inline properties set up in the xaml. So as a suggestion I'd put this vertically, which IMO is easier to read:
```xaml
<Grid 
    StyleClass="box-row" 
    IsVisible="{Binding ShowTotp}" 
    AutomationId="ItemRow">
```